ts/parts/Pointer.ts

```diff
@@ -1065,15 +1064,20 @@ class Pointer {
      */
     public onTrackerMouseOut(e: Highcharts.PointerEventObject): void {
         const chart = this.chart;
-        const series = chart.hoverSeries;
         const relatedTarget = e.relatedTarget || e.toElement;
+        const series = chart.hoverSeries;
+        const tooltip = chart.tooltip;
 
         this.isDirectTouch = false;
 
         if (
             series &&
             relatedTarget &&
             !series.stickyTracking &&
+            (
+                !tooltip ||
+                !tooltip.isStickyOnContact()
+            ) &&
```

Comment:
As above, are we mixing in some unrelated fix?

Reply:
Same here.